### PR TITLE
Update insecure.md for Docker Desktop

### PR DESCRIPTION
**Pull request by t-eimizu**
### Proposed changes

Docker Desktop does not have "preferences" menu today.
It's changed to "Settings" > "Docker Engine" .

### CODE REVIEW

File: registry/insecure.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ isolated testing or in a tightly controlled, air-gapped environment.
     `/etc/docker/daemon.json` on Linux or
     `C:\ProgramData\docker\config\daemon.json` on Windows Server. If you use
     Docker Desktop for Mac or Docker Desktop for Windows, click the Docker icon, choose
-    **Preferences**, and choose +**Daemon**.
+    **Settings**, and choose +**Docker Engine**.
```
[the remark this answers is not in the document]
Comment (StefanScherer) [reply registry/insecure.md:29]: @thaJeztah yes, it's  "Settings" on Windows and "Preferences..." on Mac. But then it's in "Docker Engine" for both platforms to edit  the `daemon.json`.

Comment (thaJeztah) [inline registry/insecure.md:29]: Thanks; perhaps then use something like;

```suggestion
    **Preferences** (Mac) or **Settings** (Windows), and choose **Docker Engine**.
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: registry/insecure.md (hunk 1 above)
Comment (t-eimizu) [reply registry/insecure.md:29]: I think it's a good idea too.

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**